shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -130,6 +130,13 @@ FLUTTER_EXPORT
  */
 - (id<FlutterPluginRegistry>)pluginRegistry;
 
+/**
+ * True if at least one frame has rendered and the ViewController has appeared.
+ *
+ * This property is reset to false when the ViewController disappears.
+ */
+@property(nonatomic, readonly) BOOL isRenderingFrames;
```

Comment:
See above. But if anything, your doc is in the past tense so this would be hasRenderedFrames. (At which point, we might as well go hasRenderedFirstFrame :D)